server/src/main/generated-java/timely/api/flatbuffer/Metric.java

```diff
@@ -0,0 +1,49 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
It relies on the flatc executable being compiled on the local machine, much like protoc.